upf/upf_forward.c

```diff
@@ -194,6 +200,24 @@ upf_forward (vlib_main_t * vm, vlib_node_runtime_t * node,
 		      goto trace;
 		    }
 		}
+	      else if (far->forward.flags & FAR_F_FORWARDING_POLICY)
+		{
+		  b->flags &= ~(VNET_BUFFER_F_OFFLOAD_TCP_CKSUM |
+				VNET_BUFFER_F_OFFLOAD_UDP_CKSUM |
+				VNET_BUFFER_F_OFFLOAD_IP_CKSUM);
```

Comment:
does that work with IPv6 ?

Usually IPv6 does not clear the VNET_BUFFER_F_OFFLOAD_IP_CKSUM flag.

Reply:
Fixed this, thanks